DataFormats/Math/src/classes_def.xml

```diff
@@ -2,9 +2,6 @@
 <selection>
   <class pattern="ROOT::Math::SMatrix<*>" />
   <class pattern="ROOT::Math::MatRepStd<*>" />
-  <class pattern="ROOT::Math::MatRepSym<*>" >
-    <field name="fOff" transient="true"/>
```

Comment:
note for posterity: this data member was removed in 2015
 https://github.com/root-project/root/commit/9219906cae4cccdf602c5452cff8a90072077ac6#diff-8e7303403181c06dc2d7e791c828076bL363